eda_leda/implementacoes/sorting_algorithms/SelectionSort.py

```diff
@@ -0,0 +1,159 @@
+                """		
+		if(leftIndex < rightIndex):
+			self.__recursiveGetMax(array,leftIndex,rightIndex,rightIndex,rightIndex)
+			self.__recursiveGetMin(array,leftIndex,rightIndex,leftIndex,leftIndex)
+			self.__recursiveBidirectionalSelectionSort(array,leftIndex + 1,rightIndex-1)
+
+	
+	def __recursiveGetMax(self,array,leftIndex,rightIndex,maxValue,index):
+		""" Método auxiliar de recursão
+		Este método pega o maior elemento do array e troca de posição com o indíce a direita
+		
+		Parametros:
+                    array: Lista a ser ordenada    
+                    leftIndex: Índice limítrofe a esquerda do array para ordenação parcial
+                    rightIndex: Índice limítrofe a direita do array para ordenação parcial
+		    maxValue: Maior valor encontrado entre os limites esquerdo e direito
+		    index: Índice do elemento a ser comparado
+		"""
+		if(index >= leftIndex):
+			if(array[maxValue] < array[index]):
+				maxValue = index
+			self.__recursiveGetMax(array,leftIndex,rightIndex,maxValue,index-1)
+		else:
+			array[maxValue],array[rightIndex] = array[rightIndex] , array[maxValue]
+
+
+	def __recursiveGetMin(self,array,leftIndex,rightIndex,minValue,index):
+                """ Método auxiliar de recursão
+                Este método pega o menor elemento do array e troca de posição com o indíce a esquerda
+                
+                Parametros:
+                    array: Lista a ser ordenada    
+                    leftIndex: Índice limítrofe a esquerda do array para ordenação parcial
+                    rightIndex: Índice limítrofe a direita do array para ordenação parcial
+                    minValue: Menor valor encontrado entre os limites esquerdo e direito
+                    index: Índice do elemento a ser comparado
+                """		
+		if(index <= rightIndex):
+			if(array[minValue] > array[index]):
+				minValue = index
+			self.__recursiveGetMin(array,leftIndex,rightIndex,minValue,index +1)
+		else:
+			array[minValue] ,array[leftIndex] = array[leftIndex] , array[minValue]
+
+	
+	def __validateParams(self,array,leftIndex,rightIndex):
+                """Verificação da validade dos parametros de limite passados
+                
+                Este método busca evitar entradas inválidas para o algoritmo de ordenação, tais como: índices negativos,
+                índice direito menor que o esquerdo, índice direito superior ao tamanho do array.
+
+                Parametros:
+                    array: Lista a ser ordenada
+                    leftIndex(Opcional): Índice limítrofe a esquerda do array para ordenação parcial
+                    rightIndex(Opcional): Índice limítrofe a direita do array para ordenação parcial
+
+                Retorna:
+                    Boolean: Indicando se os parâmetros são ou não válidos
+                """		
+		if(array == None or leftIndex < 0 or rightIndex > len(array) -1 or leftIndex >= rightIndex):
+			return False
+		else:
+			return True;
```

Comment:
Alguma razão pro ponto e vírgula aqui? 🤔 
```suggestion
			return True
```